src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -356,6 +357,22 @@ public function write($messages, $newline = false, $type = self::OUTPUT_NORMAL)
     }
 
     /**
+     * @return QuestionHelper|null
```

Comment:
I personally dislike the idea of creating a getter which can return either an object or null. It will introduce additional complexity above - the need of checking if the given result is an instance of `QuestionHelper`